operator/cmd/mesh/install.go

```diff
@@ -236,9 +240,103 @@ func InstallManifests(iop *v1alpha12.IstioOperator, force bool, dryRun bool, res
 		return iop, err
 	}
 
+	if dryRun {
+		currentObj, err := reconciler.GetObject(iop)
+		if err != nil {
+			return iop, err
+		}
+		obj, err := object.ParseYAMLToK8sObject([]byte(iopStr))
+		if err != nil {
+			return iop, err
+		}
+		expectedObj := obj.UnstructuredObject()
+		result, err := showDifferences(currentObj, expectedObj)
+		if err != nil {
+			return iop, err
+		}
+		// Display the results to the user
+		l.Print(result)
+	}
 	return iop, saveIOPToCluster(reconciler, iopStr)
 }
 
+func showDifferences(existing, target *unstructured.Unstructured) (string, error) {
+	existingYAML := []byte("")
```

Comment:
better to just declare this as var existingYAML []byte since you don't need to init.